lang/cs/CoreCLRCompatibility/AnalyzeCoreCLRCompatibility.ps1

```diff
@@ -0,0 +1,108 @@
+# Input / Output Settings
+$baseDir = "..\"
+$binDir="${baseDir}\bin\x64\Debug"
+$outputDirectory = "Output"
+
+# Compatibility Analyzer settings
+$libraryDotNetTargets = @(".NET Standard, Version=2.0")
+$exeDotNetTargets = @(".NET Core + Platform Extensions, Version=1.0", ".NET Framework, Version=4.5.1")
```

Comment:
From @markusweimer 's comments on the dev list. I'm assuming that we want 4.5.1 and NOT 4.6.1.